webapp/lib/view.dart

```diff
@@ -457,10 +457,29 @@ class ContentView {
           titleText: '$driverName',
           datasetLabels: List.filled(0, '', growable: true)
         );
+        driverChart.yUpperLimitRangeSlider.children[1].onChange.listen((e) {
+          var slider = (e.currentTarget as RangeInputElement);
+          var sliderIndicator = (e.currentTarget as Element).previousElementSibling;
+          if (int.parse(slider.value) == 0) {
+            controller.driverYUpperLimitFilters[driverName] = int.parse(slider.max);
+          } else {
+            controller.driverYUpperLimitFilters[driverName] = int.parse(slider.value);
+          }
+          controller.command(controller.UIAction.driverYUpperLimitSet, null);
+          var newValue = (int.parse(slider.value) - int.parse(slider.min)) * 100 / (int.parse(slider.max) - int.parse(slider.min));
+          var newPosition = 150 - (newValue * 2.15);
+          sliderIndicator.style.setProperty('top', 'calc(${-newValue}% + (${newPosition}px))');
```

Comment:
If possible, I'd prefer us not to use calc(). Could we compute the position in Dart, and just setting it as a static number of pixels to the css property?

Reply:
As discussed earlier, I'll create an issue to address this.